bookwyrm/migrations/0033_reviewrate.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.0.7 on 2021-01-05 05:32
+
+from django.db.models.fields.reverse_related import ManyToOneRel
+from django.db.models import Q
+from django.db import migrations, models
+import django.db.models.deletion
+
+def convert_review_rating(app_registry, schema_editor):
+    ''' take reviews with no content and turn them into ratings '''
+    db_alias = schema_editor.connection.alias
+    reviews = app_registry.get_model('bookwyrm', 'Review')
+    review_ratings = app_registry.get_model('bookwyrm', 'ReviewRating')
+    ratings = reviews.objects.using(db_alias).filter(
+        Q(content__isnull=True) | Q(content=''))
+    # replace the old review with the rating
+    for review in ratings:
+        rating = review_ratings.objects.create(
+            user=review.user, rating=review.rating, book=review.book)
+        print('-----')
+        print(rating, review)
+        for field in review._meta.get_fields():
+            if isinstance(field, ManyToOneRel) or field.name in ['status_ptr', 'id', 'remote_id']:
+                continue
+            value = getattr(review, field.name)
+            try:
+                setattr(rating, field.name, value)
```

Comment:
This all looks perfectly reasonable to me, but in my tests it isn't copying over the `name` field, and I haven't figured out why :confused:

Reply:
I'm not sure I'm understanding you/my own code correctly, but I think ratings (new form or old form( shouldn't have names set at all?